### PR TITLE
feat: add generated connect-rpc code

### PR DESCRIPTION
**Pull request by strantalis**

### CODE REVIEW

**Review by pflynn-virtru — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  // Attach to Air for debugging
```
Comment (pflynn-virtru) [inline .vscode/launch.json:2]: should this comment be in JSON

Comment (jrschumacher) [reply .vscode/launch.json:2]: Good callout. Not sure if https://github.com/air-verse/air supports JSON5

Comment (strantalis) [reply .vscode/launch.json:2]: actually maybe I should remove this and ignore all of .vscode directory

File: protocol/go/go.mod [hunk 2]
```diff
@@ -4,10 +4,11 @@ go 1.21
 
 require (
 	buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go v1.33.0-20240221180331-f05a6f4403ce.1
+	connectrpc.com/connect v1.17.0
```
Comment (dmihalcik-virtru) [inline protocol/go/go.mod:7]: maybe add 

```
go.mod linguist-generated=false
```

to the .gitattributes file? Changes to go.mod files *should* be manually reviewed

Comment (strantalis) [reply protocol/go/go.mod:7]: Didn't seem to work. I tried `protocol/go/go.mod` as well

Comment (strantalis) [reply protocol/go/go.mod:7]: I am not going to solve this in the current pr.

**Review by jakedoublev — DISMISSED** [review]

**Review by jakedoublev — APPROVED** [review]

**Pull request merged**